securing_apps/topics/oidc/java/fuse7/install-feature.adoc

```diff
@@ -6,15 +6,17 @@ You must first install the `keycloak-pax-http-undertow` and `keycloak-jaas` feat
 
 ====== Installing from the Maven Repository
 
-As a prerequisite, you must be online and have access to the Maven repository.
+.Prerequisites
+
+* you must be online and have access to the Maven repository.
```

Comment:
Should start with the upper-case? "You ...". Asking because other places start with upper-case.